src/classes/STG_PanelPaymentMapping_CTRL.cls

```diff
@@ -117,7 +117,7 @@ public with sharing class STG_PanelPaymentMapping_CTRL extends STG_Panel {
             targetTokenMap.remove(s);
         }
 
-        if ((targettokenMap.size() < 75) && (targettokenMap.size() > 0)){
+        if (targettokenMap.size() > 0){
```

Comment:
Let's use `if (!targetTokenMap.isEmpty()) {` as the more efficient way to check if the list has elements.
I am not going to ask for the class update to comply to the latest Apex style in order to minimize number of changes and testing, but if the style change was limited only to the method instead of the whole class, that would be one of the requirements for the code.

Reply:
I was thinking the same thing. The method should be updated for styling, and possibly some abstraction if it helps with readability/testing.